test-configs.yaml

```diff
@@ -839,6 +848,9 @@ test_configs:
   - device_type: alpine_db
     test_plans: [boot]
 
+  - device_type: am57xx-beagle-x15
+    test_plans: [boot, boot_nfs, kselftest, simple]
```

Comment:
Maybe `sleep` as well?

Reply:
I'll add sleep. Thanks.

Reply:
Also not sure about `kselftest`, @khilman might have some views on this.